ch07.adoc

```diff
@@ -70,7 +70,7 @@ dimensions:
 variables:
   float time(time);
     time:standard_name = "time";
-    time:units = "days since 2024-11-8 09:00:00";
+    time:units = "days since 2024-11-8 09:00:00 Z";
```

Comment:
maybe omit the space before `Z` in this one, to show that it isn't mandatory?

Reply:
done.